src/cf.data.pre

```diff
@@ -2455,7 +2455,7 @@ DOC_START
 
 	   require-proxy-header
 			Require PROXY protocol version 1 or 2 connections.
-			The proxy_protocol_access is required to whitelist
```

Comment:
I'm not sure, if this substitution is in place here. Any idea?

Reply:
LGTM